test/snapp_auction.js

```diff
@@ -169,13 +177,35 @@ contract("SnappAuction", async (accounts) => {
       const instance = await SnappAuction.new()
       await setupEnvironment(MintableERC20, instance, token_owner, [user_1], 2)
       const AUCTION_RESERVED_ACCOUNT_BATCH_SIZE = await instance.AUCTION_RESERVED_ACCOUNT_BATCH_SIZE()
-      
+      const badOrder = Buffer.from("0".repeat(AUCTION_RESERVED_ACCOUNT_BATCH_SIZE*26+1), "binary")
       await truffleAssert.reverts(
-        instance.placeStandingSellOrder(Buffer.from("0".repeat(AUCTION_RESERVED_ACCOUNT_BATCH_SIZE*26+1), "binary"), { from: user_1 }),
+        instance.placeStandingSellOrder(badOrder, { from: user_1 }),
         "Each order should be packed in 26 bytes!"
       )
     })
 
+    it("Rejects standing order requests from non-reserved accounts", async () => {
+      const instance = await SnappAuction.new()
+
+      const numReservedAccounts = (await instance.AUCTION_RESERVED_ACCOUNTS()).toNumber()
+
+      const tokens = await registerTokens(MintableERC20, instance, token_owner, 1)
+      const amount = "300000000000000000000"
```

Comment:
ubernit: Do we have to fund the account? Also 1 one order would probably be enough. (Just trying to reduce the amount of action in this test to the necessary minimum).

Reply:
I'll take a look. It is possible we don't need to fund the account.